packages/world/ts/normalizeSystemFunctionName.ts

```diff
@@ -0,0 +1,16 @@
+import { hexToResource, resourceToLabel } from "@latticexyz/common";
+import { Hex } from "viem";
+
+/** @internal */
+export function internal_normalizeSystemFunctionName(systemId: Hex, functionName: string) {
```

Comment:
I don't wanna propagate the usage of this so I prefixed this with `internal_` to make it clear